beam_conversion/cst.py

```diff
@@ -135,6 +135,8 @@ def parse_args():
 
 
 if __name__=="__main__":
+    print ("This file has not been update for work with v2 beam files. Fix first!")
```

Comment:
This is because we are only using HFSS and Feko now, right?

Reply:
Yes. I'll add support for CST when we actually have some things to use. Format still keeps changing...